docs/stoploss.md

```diff
@@ -3,74 +3,98 @@
 2. trailing stop loss, defined in the configuration.
 3. trailing stop loss, custom positive loss, defined in configuration.
+4. trailing stop loss only once the trade has reached a certain offset,
 
 !!! Note
     All stoploss properties can be configured in either Strategy or configuration. Configuration values override strategy values.
 
-Those stoploss modes can be *on exchange* or *off exchange*. If the stoploss is *on exchange* it means a stoploss limit order is placed on the exchange immediately after buy order happens successfuly. This will protect you against sudden crashes in market as the order will be in the queue immediately and if market goes down then the order has more chance of being fulfilled.
+Those stoploss modes can be *on exchange* or *off exchange*. If the stoploss is *on exchange* it means a stoploss limit order is placed on the exchange immediately after buy order happens successfully. This will protect you against sudden crashes in market as the order will be in the queue immediately and if market goes down then the order has more chance of being fulfilled.
 
-In case of stoploss on exchange there is another parameter called `stoploss_on_exchange_interval`. This configures the interval in seconds at which the bot will check the stoploss and update it if necessary. As an example in case of trailing stoploss if the order is on the exchange and the market is going up then the bot automatically cancels the previous stoploss order and put a new one with a stop value higher than previous one. It is clear that the bot cannot do it every 5 seconds otherwise it gets banned. So this parameter will tell the bot how often it should update the stoploss order. The default value is 60 (1 minute).
+In case of stoploss on exchange there is another parameter called `stoploss_on_exchange_interval`. This configures the interval in seconds at which the bot will check the stoploss and update it if necessary.
+
+For example, assuming the stoploss is on exchange, and trailing stoploss is enabled, and the market is going up, then the bot automatically cancels the previous stoploss order and puts a new one with a stop value higher than the previous stoploss order.
+The bot cannot do this every 5 seconds, otherwise it would get banned by the exchange.
+So this parameter will tell the bot how often it should update the stoploss order. The default value is 60 (1 minute).
+This same logic will reapply a stoploss order on the exchange should you cancel it accidentally.
 
 !!! Note
     Stoploss on exchange is only supported for Binance as of now.
 
 ## Static Stop Loss
 
-This is very simple, basically you define a stop loss of x in your strategy file or alternative in the configuration, which
-will overwrite the strategy definition. This will basically try to sell your asset, the second the loss exceeds the defined loss.
+This is very simple, you define a stop loss of x. This will try to sell the asset once the loss exceeds the defined loss.
```

Comment:
maybe
"a stop loss of x (as a ratio of price, i.e. x * 100% of price)."
or some detailization like this can be handy